### PR TITLE
Implemented the SumTo function as solution for the forloop exercise

### LINKED ISSUE #14: Add solution: 05_for
Goal: Implement the solution for exercise "$ex".

Acceptance criteria:
- [ ] Implement code in internal/exercises/templates/$ex
- [ ] All tests pass for $ex (go test ./...)
- [ ] Add/adjust docs or comments if needed
- [ ] Cross-link this issue in the PR description

Notes:
- Keep code idiomatic and simple.
- Include edge cases and table-driven tests where relevant.

### PR DESCRIPTION
**Pull request by bdeb1337**
## Summary

Implemented the SumTo function as solution for the forloop (05_for) exercise.

## Checklist

- [x] Tests pass: `make verify` or `golearn verify <slug>`
image
- [x] Docs updated (README/CONTRIBUTING) if needed
-> Not needed here I assume, right?
- [x] No large new dependencies

## Screenshots / Output (if CLI UX)

Paste before/after where helpful.

## Related issues

Fixes #14

### CODE REVIEW

File: internal/exercises/solutions/05_for/for.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package forloop
+
+func SumTo(n int) int {
+	sum := 0
+	for i := 1; i <= n; i++ {
+		sum += i
+	}
+	return sum
+}
```
[the remark this answers is not in the document]
Comment (bdeb1337) [reply internal/exercises/solutions/05_for/for.go:10]: Added the comment

**Review by zhravan — APPROVED** [review]
PR LGTM
